ctapipe/containers.py

```diff
@@ -180,6 +218,31 @@ class TimingParametersContainer(Container):
     )
 
 
+class CameraTimingParametersContainer(BaseTimingParametersContainer):
+    """
+    Slope and Intercept of a linear regression of the arrival times
+    along the shower main axis in the camera frame.
+    """
+
+    container_prefix = "camera_frame_timing"
+    slope = Field(
+        nan / u.m, "Slope of arrival times along main shower axis", unit=1 / u.m
+    )
+
+
+class TimingParametersContainer(BaseTimingParametersContainer):
+    """
+    Slope and Intercept of a linear regression of the arrival times
+    along the shower main axis in a
+    spherical system centered on the pointing position (TelescopeFrame)
+    """
+
+    container_prefix = "timing"
+    slope = Field(
+        nan / u.deg, "Slope of arrival times along main shower axis", unit=1 / u.deg
+    )
```

Comment:
I'd say we should make the slope in `ns/deg`, if we want this to be comparible for all cameras (the point of going to deg in the first place).  That means we do not want to have samples/deg, since the sampling frequencies are not all 1 GHz.

Reply:
Right now the peak time has no unit attached though, right?

Reply:
No (they are in samples), but the sampling frequency is available, so the parameters could still be in correct units (as I think Abelardo mentioned).   If the point of this PR is to ensure all parameters can be compared in a telescope-independent way, i.e. removing the effect of camera scale and focal length, then we should also remove the effect of sampling frequency from all timing parameters (the image itself could stay in samples), but we should multiply these parameters by `subarray.tel[tid].camera.readout.sampling_rate`

Reply:
It occurs to me that defining the slope as a velocity (deg/ns) might be a nicer unit, but that's another issue.

Reply:
> No (they are in samples), 

No, peak time is in nanoseconds! But without units attached for performance reasons. But we actually never studied the impact of using a quantity with `ns` everywhere.

Reply:
Well if this is already a Quantity, then it should be "ns/deg". We can just attach the ns in the ImageProcessor if that's easiest, but it would be good to have this unit there as it it clearly already confusing.  If it says "ns", then we know there is no need to apply the sampling rate.

Reply:
We could then also rename peak_time to peak_time_ns (if no units are there), but the *parameters* which have units already should at the very least be the correct units.

Reply:
So we can either:
- Use astropy quantities everywhere
- Hack in a "ns" to make the output consistent

plus renaming the peak time field.
I can have a look at the performance impact of using the correct unit everywhere in the code, but this might be better located in another PR?

Reply:
Yes, this should be different. Let's focus here on the transformation into telescope frame, not some other connected issues